WooCommerce/Classes/POS/Controllers/PointOfSaleOrderController.swift

```diff
@@ -157,3 +189,14 @@ extension PointOfSaleInternalOrderState: Equatable {
         }
     }
 }
+
+extension PointOfSaleOrderController {
+    enum Constants {
+        static let cashPaymentMethodID: String = "cash"
```

Comment:
Should we use `PaymentGateway.Constants.cashOnDeliveryGatewayID` "cod" or did we decide to use a different method ID and title for POS?